src/Layers/xrRender/SkeletonX.h

```diff
@@ -6,6 +6,7 @@
 #define SkeletonXH
 #pragma once
 
+#include "FVisual.h"
```

Comment:
No, this should not be here. Error is happening in `SkeletonX.cpp` and this is the file where we should include `FVisual.h`

Reply:
`CRef` was renamed to `constant`.
As for `FVisual.h`, take a look carefully, it should be in `SkeletonX.h` because of template function on line 142: `BOOL pick_bone(...)`. This function use FVisual as argument type.

Reply:
Oh, sorry. I knew that it was something like that and looked especially at that function... But somehow I didn't noticed that it takes FVisual as argument %)

Then it's reasonable to remove forward declaration of FVisual on line 16.

Reply:
Yep, and i saw, that both `SkeletonX.h` and `SkeletonX.cpp` files are include the same header file `#include "SkeletonCustom.h"`. Should i remove it from `SkeletonX.cpp`? Or leave it for now?

Reply:
Yes, remove it.